shared/js/multi_sim_action_button.js

```diff
@@ -117,11 +117,9 @@ MultiSimActionButton.prototype._updateUI = function() {
       var self = this;
       var l10nId = this._simIndication.dataset.l10nId ||
                    'gaia-sim-picker-button';
-      navigator.mozL10n.ready(function() {
```

Comment:
that doesn't matter. The code inside is only setting data-l10n-id, data-l10n-args. Everything else is async.

And nothing in l10nArgs changes when language change so re-firing it on language change doesn't make sense.

Or am I wrong here?